### PR TITLE
Fix for WFCORE-6776, Can't specify a stability level when starting a bootable JAR.

### PR DESCRIPTION
**Pull request by jfdenise**
Issue: https://issues.redhat.com/browse/WFCORE-6776

### CODE REVIEW

**Comment by yersan** [comment]
@jfdenise FYI: ManagementOnlyModeTestCase is unrelated and tracked on a different Jira https://issues.redhat.com/browse/WFLY-19219

File: bootable-jar/runtime/src/main/java/org/wildfly/core/jar/runtime/CmdUsage.java [hunk 1]
```diff
@@ -48,6 +48,9 @@ public static void init() {
         addArguments(CommandLineConstants.SECMGR);
         instructions.add(BootableJarLogger.ROOT_LOGGER.argSecurityManager());
 
+        addArguments(CommandLineConstants.STABILITY + "=<value>");
+        instructions.add(BootableJarLogger.ROOT_LOGGER.argStability());
```
Comment (yersan) [inline bootable-jar/runtime/src/main/java/org/wildfly/core/jar/runtime/CmdUsage.java:52]: Would it be possible for the Bootable JAR to get the available stability levels from ProductConfig.getStabilitySet()?

In addition, we have a kind of restriction here where the stability levels should be hidden when there is only one permissible stability level. See https://github.com/wildfly/wildfly-core/blob/b20d4c06b41542316af51ff213bb564f2574d3d3/server/src/main/java/org/jboss/as/server/CommandLineArgumentUsageImpl.java#L83-L86 as a reference. CC @pferraro

Comment (jfdenise) [reply bootable-jar/runtime/src/main/java/org/wildfly/core/jar/runtime/CmdUsage.java]: @yersan , thank-you. I adjusted to comply with what we do in the server.

**Review by yersan — APPROVED** [review]

**Comment by yersan** [comment]
Thanks @jfdenise

**Pull request merged**